Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -187,13 +187,21 @@ CGMutablePathRef CGPathCreateMutable() {
  @Notes Creates a mutable copy
 */
 CGPathRef CGPathCreateCopy(CGPathRef path) {
+    if( path == NULL ) {
```

Comment:
>( [](start = 6, length = 1)

the formatting of all the if statements doesnt follow the conventions.  Can you please make sure you have run clang formatter?  (ex: it should be if (path == NULL) instead of if( path == NULL )